customisation/glossaries.md

```diff
@@ -5,3 +5,45 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
 ---
+#Glossaries
+
+Glossaries are lists of terms in one language aligned to their translation in one or more languages. They can be added to machine translation systems.
+Adding a glossary to a commercial machine translation provider can increase the terminology accuracy of the output.
+
+## Application
+
+Glossaries are leveraged at translation time, and no training is required.
+When a glossary term is matched in the source sentence, its translation is enforced in the output.
+
+The insertion of a glossary term in the output happens regardless of the context.
+The terms included in the glossary should therefore be unambiguous and context-independent, as in the following examples:
+- Respiratory syndrome
+- Balance sheet
+- Paraffin
+
+Ambiguous terms can have a negative impact on the output:
+- Spread
+- Cookies
+- Surface treatment
+
+##Term matching and insertion
+
+Several machine translation providers offer the possibility to enhance their output using a glossary.
+The way in which the glossary is leveraged may differ in terms of source term matching and target term insertion.
+
+In some cases the source term is matched in the source text only if it occurs with the same morphology and ortography as in the glossary.
+Some providers match the source term regardless of the morphology and ortography of its occurrence in the source sentence.
+
+The target term can be inserted either with the same form as in the glossary, or by adapting the term to the rest of the sentence and viceversa, for example ensuring a correct agreement between the subject and the verb.
+
+## Glossary term forms
+
+If the source and target terms are used with the same form as in the glossary, the latter should cover the following forms, depending on the language:
```

Comment:
I think this section a bit too much detail, and it's very related to the previous section.

We could just replace it all with:

> With strict exact matching and insertion, more variants need to be listed in the glossary.

Or

> With matching and insertion that handles common variants, multiple variants of the same term do not need to be listed in the glossary.

What do you think?

Reply:
Agreed!